src/content/docs/solvro/Strategy.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Strategia KN Solvro
+sidebar:
+  order: 1
+description: Długoterminowy plan strategiczny KN Solvro
+---
+
+import {
+  Steps,
+  Card,
+  CardGrid,
+  LinkCard,
+  Aside,
+} from "@astrojs/starlight/components";
+
+> ### <i>"Otwieramy drzwi do świata IT" ~ motto KN Solvro</i>
+
+## Początki
+
+W 2018 roku grupa studentów napędzona kreatywnością i chęcią tworzenia założyła **Koło Naukowe Solvro**. Od tego momentu, przez te lata nasza organizacja działała, podejmując coraz to nowsze inicjatywy. Zarządy się zmieniały, a wraz z nimi początkowa wizja zanikała.
+
+W 2024 roku, po objęciu władzy przez VII zarząd, nastąpiła restrukturyzacja. Pół roku później, w wyniku wyjazdu do Lubawki, powstał zarys działań. Pod koniec 2024 roku powstał ten dokument, czyli **LTSP** (Long Term Strategic Plan) mający na celu wytyczenie ścieżki dla kolejnych zarządów, przekładając wiedzę obecnego pokolenia członków na wartościowe wnioski zapisane w tym planie.
+
+**KN Solvro** ma przed sobą świetlaną przyszłość, ale to właśnie rolą kolejnych generacji członków i zarządów jest pielęgnowanie i rozwijanie tej przyszłości 🚀.
+
+---
+
+## Nasze fundamenty
+
+Wywodząc się z Politechniki Wrocławskiej utrzymujemy jej kierunek wyznaczając własną misję i wizję.
+
+### Misja
+
+> ##### „Jesteśmy mostem między teorią i praktyką, kształcąc programistów napędzających innowacje na Politechnice Wrocławskiej.”
```

Comment:
Tak sobie myślałem, czy nie zmienić 'programistów' na '(kolejne) pokolenia'. Wiem, że sam tutaj dodawałem 'programistów', natomiast nasza działalność zaczęła już wykraczać poza ten label. No, będzie to też wymagało zmiany w paru innych miejscach xd

Reply:
(ja też mogę być uwu)

Reply:
Hmm, ciekawe zagadnienie. Jak chcemy dać kolejne pokolenia [to tu coś musimy wstawić] 

Może kolejne pokolenia innowatorów, ale już mamy innowacje w słowa dalej
Kolejne pokolenia członków brzmi średnio
Kolejne pokolenia studentów?